src/AWS.Logger.Log4net/AWSAppender.cs

```diff
@@ -44,6 +44,22 @@ public bool DisableLogGroupCreation
             set { _config.DisableLogGroupCreation = value; }
         }
 
+        /// <summary>
+        /// Specifies the days to retain events in log groups which are created by this logger, if the one specified by <see cref="LogGroup"/> doesn't already exist
+        /// and <see cref="DisableLogGroupCreation"/> is not <c>true</c>. Requires logs:PutRetentionPolicy permission to apply the retention policy to
+        /// newly created log groups. The default value of <c>null</c> will not apply a retention policy to new log groups.
+        /// <para>
+        /// Note that log groups which already exist will not have this retention policy applied for startup performance reasons.
+        /// </para>
+        /// <para>
+        /// Possible values could be found in the CloudWatchLogs API reference at https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutRetentionPolicy.html#API_PutRetentionPolicy_RequestSyntax
```

Comment:
@Will-Boulton Thanks for addressing review comments and rebasing commits. However, I still see the extra URL in the code comment. Link https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutRetentionPolicy.html#API_PutRetentionPolicy_RequestSyntax is mentioned twice on separate lines. Could you please check?

Reply:
Well spotted, fixed